aspnetcore/release-notes/aspnetcore-7.0.md

```diff
@@ -209,6 +209,266 @@ SignalR hub methods now support injecting services through dependency injection
 
 Hub constructors can accept services from DI as parameters, which can be stored in properties on the class for use in a hub method. For more information, see [Inject services into a hub](xref:signalr/hubs?view=aspnetcore-7.0&preserve-view=true#inject-services-into-a-hub)
 
+## Blazor
+
+### Handle location changing events
+
+In .NET 7, Blazor supports location changing events. This allows you to warn users about unsaved work or to perform related actions when the user performs a page navigation.
+
+For more information, see the following sections of the *Routing and navigation* article:
+
+* [Navigation options](xref:blazor/fundamentals/routing?view=aspnetcore-7.0#navigation-options)
+* [Handle/prevent location changes](xref:blazor/fundamentals/routing?view=aspnetcore-7.0#handleprevent-location-changes)
+
+### Empty Blazor project templates
+
+Blazor has two new project templates for starting from a blank slate. The new **Blazor Server App Empty** and **Blazor WebAssembly App Empty** project templates are just like their non-empty counterparts but without demonstration code. These empty templates only include a basic home page, and we've removed Bootstrap so that you can start with a different CSS framework.
+
+For more information, see the following articles:
+
+* <xref:blazor/tooling?view=aspnetcore-7.0>
+* <xref:blazor/project-structure?view=aspnetcore-7.0>
+
+### Blazor custom elements
+
+The [`Microsoft.AspNetCore.Components.CustomElements`](https://www.nuget.org/packages/microsoft.aspnetcore.components.customelements) package enables building [standards based custom elements](https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements) using Blazor.
+
+For more information, see <xref:blazor/components/index?view=aspnetcore-7.0#blazor-custom-elements>.
+
+<!--
+
+    IN PROGRESS
+
+### Improve prerendering and authentication experience in Blazor
+
+[27592](https://github.com/dotnet/aspnetcore/issues/27592)
+
+-->
+
+### Bind get/set/after modifiers
+
+New modifiers, `@bind:get`/`@bind:set`/`@bind:after`, allow you to customize how Blazor's `@bind` attribute directive reads and writes bound values.
+
+<!--
+
+    NOTE
+    
+    `@bind:get`/`@bind:set` are only covered in an example for binding across more than two components. 
+    There isn't textual coverage outside of that section or dedicated examples for these.
+
+-->
+
+For more information, see the following content in the *Data binding* article:
+
+* [`@bind:after="{EVENT}"` content](xref:blazor/components/data-binding?view=aspnetcore-7.0) at the end of the article's introductory guidance.
+* The [Bind across more than two components](xref:blazor/components/data-binding?view=aspnetcore-7.0#bind-across-more-than-two-components) section demonstrates the use of `@bind:get`/`@bind:set` syntax.
+
+Additional information:
```

Comment:
I'd rather not point to point-in-time design notes. We should just make sure we have these new modifiers appropriately covered in the Blazor data binding docs.

Reply:
I only have coverage for `@bind:after`. I was going to base the doc coverage on your blog, but I don't think you ever explained them out. I can attempt to enhance from the PU issue/PR. For now, I'll remove the additional links.

Reply:
I've made a note on this. I'll work on it further separately on Monday/Tuesday, then I'll return here to cross-link new content for `get`/`set`.

Reply:
> I was going to base the doc coverage on your blog, but I don't think you ever explained them out.

OK, let me know how I can help out here with the get/set content.